react-frontend/src/components/GroupServiceFailures.jsx

```diff
@@ -132,6 +132,21 @@ const GroupServiceFailures = ({ group = null, service, collapse }) => {
 
   if (loaded) return <ServiceLoader text="Loading series.." />;
 
+  const generateTooltipPosition = (...args) => {
+    const position = args?.[0]; //tooltip default position
+    const place = args?.[4]; // tooltip placement
+    const offset = 6;
+
+    if (place === "left" || place === "right") {
```

Comment:
we can use `['left', 'right'].includes(place)`

Reply:
?